Sofa/Component/Constraint/Lagrangian/Model/src/sofa/component/constraint/lagrangian/model/BilateralInteractionConstraint.inl

```diff
@@ -98,19 +94,14 @@ template<class DataTypes>
 void BilateralInteractionConstraint<DataTypes>::reinit()
 {
     prevForces.clear();
-    activated = (activateAtIteration.getValue() >= 0 && activateAtIteration.getValue() <= iteration);
 }
 
-template<class DataTypes>
-void BilateralInteractionConstraint<DataTypes>::reset(){
-    init();
```

Comment:
yes this was causing the double init. Because reset is called at start !!!

```
Sofa.Component.StateContainer.dll!sofa::component::statecontainer::MechanicalObject<sofa::defaulttype::StdRigidTypes<3,double>>::reset() Line 1181	C++
 	Sofa.Simulation.Core.dll!sofa::simulation::ResetVisitor::processNodeTopDown(sofa::simulation::Node * node) Line 39	C++
 	Sofa.Simulation.Graph.dll!sofa::simulation::graph::DAGNode::executeVisitorTopDown(sofa::simulation::Visitor * action, std::list<sofa::simulation::graph::DAGNode *,std::allocator<sofa::simulation::graph::DAGNode *>> & executedNodes, std::map<sofa::simulation::graph::DAGNode *,sofa::simulation::graph::DAGNode::StatusStruct,std::less<sofa::simulation::graph::DAGNode *>,std::allocator<std::pair<sofa::simulation::graph::DAGNode * const,sofa::simulation::graph::DAGNode::StatusStruct>>> & statusMap, sofa::simulation::graph::DAGNode * visitorRoot) Line 688	C++
 	Sofa.Simulation.Graph.dll!sofa::simulation::graph::DAGNode::executeVisitorTopDown(sofa::simulation::Visitor * action, std::list<sofa::simulation::graph::DAGNode *,std::allocator<sofa::simulation::graph::DAGNode *>> & executedNodes, std::map<sofa::simulation::graph::DAGNode *,sofa::simulation::graph::DAGNode::StatusStruct,std::less<sofa::simulation::graph::DAGNode *>,std::allocator<std::pair<sofa::simulation::graph::DAGNode * const,sofa::simulation::graph::DAGNode::StatusStruct>>> & statusMap, sofa::simulation::graph::DAGNode * visitorRoot) Line 698	C++
 	Sofa.Simulation.Graph.dll!sofa::simulation::graph::DAGNode::doExecuteVisitor(sofa::simulation::Visitor * action, bool precomputedOrder) Line 607	C++
 	[Inline Frame] Sofa.Simulation.Core.dll!sofa::simulation::Node::execute(const sofa::core::ExecParams *) Line 206	C++
 	Sofa.Simulation.Core.dll!sofa::simulation::Simulation::reset(sofa::simulation::Node * root) Line 289	C++
 	Sofa.GUI.Qt.dll!sofa::gui::qt::RealGUI::resetScene() Line 2371	C++
>	Sofa.GUI.Qt.dll!sofa::gui::qt::RealGUI::setSceneWithoutMonitor(boost::intrusive_ptr<sofa::simulation::Node> root, const char * filename, bool temporaryFile) Line 1076	C++
 	Sofa.GUI.Qt.dll!sofa::gui::qt::RealGUI::setScene(boost::intrusive_ptr<sofa::simulation::Node> root, const char * filename, bool temporaryFile) Line 1088	C++
 	Sofa.GUI.Common.dll!sofa::gui::common::GUIManager::SetScene(boost::intrusive_ptr<sofa::simulation::Node> groot, const char * filename, bool temporaryFile) Line 274	C++
 	runSofa.exe!main(int argc, char * * argv) Line 493	C++

```